### DISCUSSION TITLE
Zoom with different scales/ranges

### DISCUSSION THREAD

**Discussion by RubenZx** [nolimits4web/swiper#4727] [Q&A]
Is there any way to zoom.in() and zoom.out() at different levels of zoom, instead of have zoom (zoom.in) or not have zoom (zoom.out)? The problem is that I want different levels of zoom, like 100%-150%-200%-250%-300% (100% = minRatio, 300% = maxRatio), when I'm zooming, and not to get the maxRatio zooming when I zoom.in(), I would like to be able to go by steps.

I've found this solution: [Zoom in on a mousewheel point (using scale and translate)](https://stackoverflow.com/questions/46647138/zoom-in-on-a-mousewheel-point-using-scale-and-translate) using js + css, but I think that this feature should be included in the library, instead of doing it by hand.

**Comment by 3zzy** [comment]
Yes please, really need this!

**Reply by 3zzy** [reply]
Wondering if it has been implemented yet?

**Comment by valyaevadaria** [comment]
I need this too!
